pom.xml

```diff
@@ -11,7 +11,8 @@
     <packaging>pom</packaging>
     <name>${project.groupId}:${project.artifactId}</name>
     <description>
-        The belgif-rest-guide-validator Maven plugin is used to validate if an OpenAPI document conforms to the guidelines in the Belgif REST guide
+        The belgif-rest-guide-validator Maven plugin is used to validate if an OpenAPI document conforms to the
```

Comment:
you can remove "Maven plugin" here, we can document the choice between maven and cli in the README.MD